vue-app/src/router/index.ts

```diff
@@ -66,6 +69,21 @@ const routes = [
     name: 'join',
     component: JoinLanding,
   },
+  {
+    path: '/setup',
+    name: 'setup',
+    component: SetupLanding,
+  },
+  {
+    path: '/setup/get-verified/:step',
```

Comment:
@ryancreatescopy curious on your reasoning for /get-verified/ subroute - is the thinking there will be other /setup/ routes, e.g. for l2?